hf_olmo/convert_olmo_to_hf.py

```diff
@@ -91,6 +92,14 @@ def download_remote_checkpoint_and_convert_to_hf(checkpoint_dir: str, local_dir:
     return local_model_path
 
 
+def fix_bad_tokenizer(checkpoint_dir: str):
+    path = os.path.join(checkpoint_dir, "config.yaml")
+    conf = om.load(path)
+    conf["tokenizer"]["identifier"] = "allenai/gpt-neox-olmo-dolma-v1_5"
```

Comment:
Just checking, was the tokenizer also wrong for the OLMo models we released, or just for the 1.7 runs? I just want to make sure that this script doesn't break old checkpoints, or that we warn that there is such a risk.

Reply:
I ran this explicitly for the v1 models before releasing.